flyteidl/protos/flyteidl/plugins/tensorflow.proto

```diff
@@ -6,9 +6,10 @@ option go_package = "github.com/flyteorg/flyte/flyteidl/gen/pb-go/flyteidl/plugi
 
 // Custom proto for plugin that enables distributed training using https://github.com/kubeflow/tf-operator
 message DistributedTensorflowTrainingTask {
-  // number of worker, ps, chief replicas spawned in the cluster for this job
+  // number of worker, ps, chief and evaluator replicas spawned in the cluster for this job
```

Comment:
nit: can you move the comments to each field?

Reply:
No problem.